components/HomepageSlider/HomepageSlider.tsx

```diff
@@ -0,0 +1,157 @@
+import { useState, useEffect } from 'react';
+import { useOnWindowResize } from '../../custom_hooks/useOnWindowResize';
+import styles from "./HomepageSlider.module.scss";
+import Slide from './Slide';
+
+export default function HomepageSlider() {
+    const window_resize = useOnWindowResize();
+    const NEXT = 'next';
+    const PREVIOUS = 'previous';
+    const LAST_SLIDE = 'last_slide';
+    const FIRST_SLIDE = 'first_slide';
+    const slide_amount = 4; // starts at 0
+    const slide_size = { small: 70, big: 33 };
+    const [is_holding_card, setIsHoldingCard] = useState(false);
+    const [in_slide_transition, setInSlideTransition] = useState(false);
+    const [enable_slide_animation, setEnableSlideAnimation] = useState(true);
+    const [slide_distance, setSlideDistance] = useState(0);
+    const [mouse_down_position, setMouseDownPosition] = useState(0);
+    const [slide_position, setSlidePosition] = useState(1);
+    
+    useEffect(() => {
+        if (window_resize.width >= 768) {
```

Comment:
Good practice to define the screen sizes and reused them all over the code.
example: devices.scss in deriv app

Reply:
I agree, good point. I added a solution for that now.